### DISCUSSION TITLE
File not found: scalex-tutorial/pancreas.h5ad

### DISCUSSION THREAD

**Discussion by vivek-ksingh** [jsxlei/SCALEX#22] [Q&A] [answered]
Hello SCALEX team,

I am not able to download the pancreas data mentioned in Projection_pancreas.ipynb tutorial is not available. It throws an error: "ERROR 404: File not found." (see screenshot below).

Can you please point me to the correct location from where I can download these files?

Thanks,
Vivek

<img width="741" alt="image" src="https://github.com/jsxlei/SCALEX/assets/60358916/581b49b8-d7e1-4345-af4c-158906720cd1">

**Comment by jsxlei** [comment, marked as answer]
Apologies for the inconvenience with the data download. Please find below the alternative links to download the files.
https://www.dropbox.com/scl/fi/dnwpv29kcfl449a8aikqi/pancreas.h5ad?rlkey=9me8xa0n1f3lz8qqornkh0ae0&dl=0
https://www.dropbox.com/scl/fi/obdt0ecjjlis6tyekyzyy/pancreas_query.h5ad?rlkey=cwik9m5kqrookmi84vdmwhnqk&dl=0

Let me know if you have any questions.

**Reply by vivek-ksingh** [reply]
Thanks for sharing the data files. Appreciate it very much.

Best,
Vivek
